src/templates/packages/tamagui/App.tsx.ejs

```diff
@@ -0,0 +1,11 @@
+import "react-native-gesture-handler";
+import { TamaguiProvider } from 'tamagui'
+import RootStack from "./src/navigation";
```

Comment:
@alitnk this seems like it wouldn't work if you used Expo Router and Tamagui, no?

Reply:
24383898617f83699cba1aa3e90f67092c3ba804